tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "baseUrl": ".",
     "paths": {
       "protractor": ["test/protractor.d.ts"]
-    }
+    },
+    "esModuleInterop": true
```

Comment:
Are we sure we want to add this in this?
I'm not a big typescript expert, but this might have a big effect on the lib, no?
I think you can import lodash with `*` instead, just for now.
i.e.:
```js
import * as isNumber from 'lodash/isNumber';
```

Reply:
@zhukovka 
Did you see my comment here?

Reply:
sorry, I missed it 
my TS doesn't accept 
` import * as isNumber from 'lodash/isNumber'; `